src/Microsoft.Azure.WebJobs.Host/Loggers/Logger/LoggerConfiguration.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Host.Loggers
+{
+    /// <summary>
+    /// Provides a filter for use with an <see cref="ILogger"/>.
+    /// </summary>
+    [System.CLSCompliant(false)]
+    public class FilterBuilder
```

Comment:
Yeah, good call. I pulled this out into a class pretty late and didn't give the name much thought. How about `CategoryFilter`? And I can change the `CategoryFilters` property to `Levels` (or `CategoryLevels`?)